test/src/main/java/org/apache/accumulo/test/manager/SuspendedTabletsIT.java

```diff
@@ -140,6 +193,7 @@ public void shutdownAndResumeTserver() throws Exception {
       }
 
       // remove servers with metadata on them from the list of servers to be shutdown
+      assertEquals("Expecting a single tServer in metadataServerSet", 1, metadataServerSet.size());
       tserverSet.removeAll(metadataServerSet);
 
       assertEquals("Expecting two tServers in shutdown-list", 2, tserverSet.size());
```

Comment:
Could add another assert around this line to ensure `tserverSet.size() >= count`